test/src/test/java/jenkins/model/JenkinsManagePermissionTest.java

```diff
@@ -226,4 +239,51 @@ public void describeMismatch(Object item, Description description) {
 
     // End of HudsonTest
     //-------
+
+    @Issue("JENKINS-63795")
+    @Test
+    public void managePermissionShouldBeAllowedToRestart() throws IOException {
+
+        //GIVEN a Jenkins with 3 users : ADMINISTER, MANAGE and READ
+        HudsonPrivateSecurityRealm realm = new HudsonPrivateSecurityRealm(false, false, null);
+        User adminUser = realm.createAccount("Administer", "G0d");
+        User manageUser = realm.createAccount("Manager", "TheB00S");
+        User readUser = realm.createAccount("Reader", "BookW00rm");
+        j.jenkins.setSecurityRealm(realm);
+
+        ProjectMatrixAuthorizationStrategy authorizationStrategy = new ProjectMatrixAuthorizationStrategy();
+        authorizationStrategy.add(Jenkins.ADMINISTER, adminUser.getId());
+
+        authorizationStrategy.add(Jenkins.MANAGE, manageUser.getId());
+        authorizationStrategy.add(Jenkins.READ, manageUser.getId());
+
+        authorizationStrategy.add(Jenkins.READ, readUser.getId());
+        j.jenkins.setAuthorizationStrategy(authorizationStrategy);
```

Comment:
Nit: Much easier with `j.createDummySecurityRealm` (all users with username=password exist) and `MockAuthorizationStrategy` which has a fluent API for granting permissions.

Reply:
ok thanks :)